crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -971,3 +971,31 @@ pub fn transpose_config(curr: &Node) -> Vec<i64> {
 
     perm
 }
+
+pub fn squeeze_config(curr: &Node) -> Vec<i64> {
+    for (key, value) in curr.attrs.iter() {
+        match key.as_str() {
+            "axes" => return value.clone().into_i64s(),
+            _ => {}
```

Comment:
Great! This will support [ONNX OPset 13 ](https://onnx.ai/onnx/operators/onnx__Squeeze.html#squeeze-13)

Can we, to be sure it works, add a unit test for OPset 13? This is similar to unsqueeze with opset 16 and 13.

Reply:
Unsqueeze uses OPset 16 and 11. I think my original one used 16, so I added one for 13. Is that what you meant?

Reply:
Yes, both should be good. OpSet 13 has `axes` attribute so it should work.